src/plugins/simulator/fs9/simulatorfs9.cpp

```diff
@@ -194,6 +194,15 @@ namespace BlackSimPlugin::Fs9
         return CSimulatorFsCommon::physicallyRemoveRemoteAircraft(callsign);
     }
 
+    bool CSimulatorFs9::dontOverwrite(const CFrequency &from, const CFrequency &to)
+    {
+        bool ergeb = false;
+        // don't overwrite 8.33 kHz frequencies
+        ergeb = (CComSystem::isExclusiveWithin8_33kHzChannel(from)); //&& from.valueInteger(CFrequencyUnit::kHz()) == 132200);
+        // && from.valueInteger(CFrequencyUnit::kHz()) / 10 == to.valueInteger(CFrequencyUnit::kHz()) / 10;
```

Comment:
Is this code commented out intentionally?

Reply:
function completely removed